docs/usage/configuration-options.md

```diff
@@ -742,6 +742,17 @@ A similar one could strip leading `v` prefixes:
 
 Set this to `false` if you want to disable release notes fetching.
 
+Renovate can fetch release notes when you're running it on one of these platforms:
+
+- GitHub (.com and Enterprise)
+- GitLab (.com and CE/EE)
```

Comment:
With this write up, it remains unclear what I need to do to e.g. my docker images to get Renovate to look up their release notes.

Reply:
I don't think that's relevant for this particular section "fetchReleaseNotes true/false"

Reply:
> With this write up, it remains unclear what I need to do to e.g. my docker images to get Renovate to look up their release notes.

@astellingwerf I'm not sure what you mean here. Do you mean:

1. I don't know how to get Renovate to show Docker image release notes.
2. I don't know how to get Renovate to look up any release notes when I'm running Renovate with Docker.

Reply:
I've created issue #15052 to discuss:

> With this write up, it remains unclear what I need to do to e.g. my docker images to get Renovate to look up their release notes.

I'm resolving this conversation, so the PR can get merged. 😉